### PR TITLE
[#6094]: Persist extensions

### LINKED ISSUE #6094: Persist extensions
### User Story

As a user, I want my extensions to persist across restarts.

### Description

*Type*: Feature

After an extension is installed (part of the app-db) it should be persisted and reactivated upon restarts, based on its state.

TBD: do all extensions need to be persisted? What is the default behavior then?

### PR DESCRIPTION
**Pull request by alwx**
Fixes #6094

Also includes some very minor refactoring of extensions events

### CODE REVIEW

File: src/status_im/extensions/core.cljs [hunk 1]
```diff
@@ -43,3 +47,64 @@
...
+  {:db (update db :extensions/manage assoc input-key {:value value})})
+
+(fx/defn fetch [cofx id]
+  (get-in cofx [:db :account/account :extensions id]))
+
+(fx/defn edit
+  [cofx id]
+  (let [{:keys [url]} (fetch cofx id)]
+    (-> (set-input cofx :url (str url))
+        (assoc :dispatch [:navigate-to :edit-extension]))))
```
Comment (janherich) [inline src/status_im/extensions/core.cljs]: Should be `navigation/navigate-to` call instead of unnecessary dispatch

**Review by janherich — CHANGES_REQUESTED** [review]

**Comment by alwx** [comment]
Rebasing is done!

**Review by jeluard — APPROVED** [review]

File: src/status_im/extensions/core.cljs [hunk 2]
```diff
@@ -183,3 +186,63 @@
...
+  [{:keys [db]} input-key value]
+  {:db (update db :extensions/manage assoc input-key {:value value})})
+
+(fx/defn fetch [cofx id]
+  (get-in cofx [:db :account/account :extensions id]))
+
+(fx/defn edit
+  [cofx id]
+  (let [{:keys [url]} (fetch cofx id)]
+    (fx/merge (set-input cofx :url (str url))
```
Comment (janherich) [inline src/status_im/extensions/core.cljs:200]: I'm afraid you forgot to pul `cofx` as first argument to `fx/merge`

**Review by janherich — CHANGES_REQUESTED** [review]

File: src/status_im/extensions/core.cljs (hunk 2 above)
Comment (alwx) [reply src/status_im/extensions/core.cljs:200]: `(set-input cofx :url (str url))` doesn't count? At least it works as expected.

**Review by janherich — APPROVED** [review]

**Comment by asemiankevich** [comment]
when installing extension
image

**Comment by asemiankevich** [comment]
okay it is fixed now. lets merge

**Pull request merged**